src/main/java/com/github/manolo8/darkbot/gui/plugins/UpdateProgressBar.java

```diff
@@ -0,0 +1,11 @@
+package com.github.manolo8.darkbot.gui.plugins;
+
+import javax.swing.*;
+
+public class UpdateProgressBar extends JProgressBar {
+    public UpdateProgressBar() {
+        super();
```

Comment:
```suggestion
```
If this is barely used, make sure it is needed, if all you do is set 2 values, may as well avoid subclassing